README.md

```diff
@@ -1,8 +1,12 @@
 # Doctor
 
-[![Hex.pm](https://img.shields.io/hexpm/v/doctor.svg)](http://hex.pm/packages/doctor) [![Build
-Status](https://travis-ci.org/akoutmos/doctor.svg?branch=master)](https://travis-ci.org/akoutmos/doctor) [![Coverage
-Status](https://coveralls.io/repos/github/akoutmos/doctor/badge.svg?branch=master)](https://coveralls.io/github/akoutmos/doctor?branch=master)
+[![Build Status](https://travis-ci.org/akoutmos/doctor.svg?branch=master)](https://travis-ci.org/akoutmos/doctor)
```

Comment:
Switch the list around so that the module version is the first thing.

Reply:
> Switch the list around so that the module version is the first thing.

Done. :smile: